`stub_heatMap.cpp` clang-diagnostic-errors

Haven't looked at it yet, but since recently, `clang-tidy` had issues with heatMap. Maybe some header missing ?

@osamahammad21,  you recently modified this file, did you encounter anything that could explain it ? 

```
src/gui/src/stub_heatMap.cpp:38:7: error: member initializer 'renderer_' does not name a non-static data member or base class [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:39:7: error: member initializer 'setup_' does not name a non-static data member or base class [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:196:1: error: use of undeclared identifier 'HeatMapRenderer' [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:201:6: error: use of undeclared identifier 'HeatMapRenderer' [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:205:13: error: use of undeclared identifier 'HeatMapRenderer' [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:210:20: error: use of undeclared identifier 'HeatMapRenderer' [clang-diagnostic-error]
src/gui/src/stub_heatMap.cpp:215:6: error: use of undeclared identifier 'HeatMapRenderer' [clang-diagnostic-error]
```

--- a/src/gui/src/stub_heatMap.cpp
+++ b/src/gui/src/stub_heatMap.cpp
@@ -35,6 +35,7 @@ HeatMapDataSource::HeatMapDataSource(utl::Logger* logger,
       reverse_log_(false),
       show_numbers_(false),
       show_legend_(false),
+      use_selected_only_(false),
       color_generator_(SpectrumGenerator(100.0))
 {
 }

